overlord/configstate/configcore/kernel_test.go

```diff
@@ -474,3 +492,26 @@ func (s *kernelSuite) TestConfigureKernelCmdlineSignedGradeNotAllowed(c *C) {
 		s.testConfigureKernelCmdlineSignedGradeNotAllowed(c, cmdline)
 	}
 }
+
+func (s *kernelSuite) TestConfigureKernelCmdlineOnClassicFails(c *C) {
+	s.mockClassicBasicModelWithoutSnaps()
```

Comment:
it may be useful to have a separate test for the hybrid setup, and do we already have a spread test for the hybrid scenario?

Reply:
The fde-on-classic spread test checks this for hybrid systems